components/automate-ui/src/app/pages/automate-settings/automate-settings.component.html

```diff
@@ -158,7 +159,8 @@ <h2>Client Runs</h2>
             <div class="checkbox-container">
               <chef-checkbox
               [checked]="!clientRunsRemoveNodes.value.disabled"
-              (change)="handleFormActivation(clientRunsRemoveNodes, $event.detail)">
+              (change)="handleFormActivation(clientRunsRemoveNodes, $event.detail)"
+              disabled="isDesktopView">
```

Comment:
The checkboxes had to be disabled in that here in the template. The other controls could not be disable in the template because of the below warning that I was seeing. 
```
It looks like you’re using the disabled attribute with a reactive form directive. If you set disabled to true when you set up this control in your component class, the disabled attribute will actually be set in the DOM for you. We recommend using this approach to avoid ‘changed after checked’ errors.
```
https://netbasal.com/disabling-form-controls-when-working-with-reactive-forms-in-angular-549dd7b42110

Reply:
I often forget when it does/doesn't need it.  When testing, looks like disabled needs square brackets.  Sorry about that!

`[disabled]="isDesktopView"`

otherwise the value remains as 'false' for the checkbox.  Same with the other one.